### ISSUE TITLE
Don't load tabs before select them

### ISSUE THREAD

**Issue by YandLiu** [cnwangjie/better-onetab#155]
Just similar to how the extension "Session Boss" or "SimpleBrowsingSchedule" behave.

**Comment by ArchonFine** [comment]
Yes, when you open a tab list with many Youtube videos in it, they all start to autoplay, making you to close and reopen the window as well as experience a huge performance hit.
